config/application.rb

```diff
@@ -29,6 +29,9 @@ class Application < Rails::Application
     # Don't generate system test files.
     config.generators.system_tests = nil
 
+    config.generators do |g|
+      g.orm :active_record, primary_key_type: :uuid
+    end
```

Comment:
Might be worth having a comment for this, especially given that most of the other stuff in the file has a comment.

Reply:
Fair enough, i'll put a comment in.

Reply:
I have had to update most of the test due to conflicting changes in master branch